src/components/ChatLog.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+// import chatMessages from '../data/messages.json';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  return (
+    <div className="chat-log">
+      {props.entries.map((entry, i) => {
+        return (
+          <ChatEntry
+            key={entry.id}
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            liked={entry.liked}
+            onLike={props.onLike}
+          />
+        );
+      })}
+    </div>
+  );
+};
+
+ChatLog.propTypes = {
+  entries: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      sender: PropTypes.string.isRequired,
+      body: PropTypes.string.isRequired,
+      timeStamp: PropTypes.string.isRequired,
+      liked: PropTypes.bool.isRequired,
+    })
+  ),
```

Comment:
If we wanted to require the entries prop, we could add a '.isRequired' here as well!